go/vt/topo/srv_vschema.go

```diff
@@ -134,3 +138,78 @@ func (ts *Server) DeleteSrvVSchema(ctx context.Context, cell string) error {
 	nodePath := SrvVSchemaFile
 	return conn.Delete(ctx, nodePath, nil)
 }
+
+// RebuildVSchema rebuilds the SrvVSchema for the provided cell list
```

Comment:
With the change to move the logic to rebuild the vschema into vtctl, do we still need to move RebuildVSchema from topotools into the server itself?

Reply:
I would prefer to keep this as it was a simple refactor and it makes more sense to have this as a member function of topo and not part of topotools. I think there may have been concerns with import loops, but by moving the test here to topotest I was able to resolve the import loop